### PR TITLE
rand method for NIG

### PR DESCRIPTION
**Pull request by robertdj**

### CODE REVIEW

**Comment by jmxpearson** [comment]
It's a bit unclear. There are tests for samplers, where some basic stats inequalities with low probabililty of being violated are checked (for instance, that the mean of 10^5 random samples is not too far from expectation); there are tests for some specialized distributions like `MvNormal` and `MvLogNormal` that involve lots of random draws. 

But I don't see much instrumentation in the test suite for testing `rand` in the case of univariates. @andreasnoack, thoughts?

**Comment by robertdj** [comment]
Any news with this issue? Is there something I can check locally?

File: src/univariate/continuous/normalinversegaussian.jl [hunk 1]
```diff
@@ -56,3 +56,26 @@ function logpdf(d::NormalInverseGaussian, x::Real)
...
+# mixture with an inverse Gaussian as mixing distribution (albeit with a
+# different parametrization than the one in Distributions).
+#
+# Ole E. Barndorff-Nielsen (1997)
+# Normal Inverse Gaussian Distributions and Stochastic Volatility Modelling
+# Scandinavian Journal of Statistics, Vol. 24, pp. 1--13
+# DOI: http://dx.doi.org/10.1111/1467-9469.00045
+
+function Distributions.rand(d::NormalInverseGaussian)
+	μ, α, β, δ = params(d)
```
Comment (ararslan) [inline src/univariate/continuous/normalinversegaussian.jl]: Please use 4 spaces for indentation rather than tabs.

Comment (robertdj) [inline src/univariate/continuous/normalinversegaussian.jl]: Sure.

**Comment by matbesancon** [comment]
Closed for #909

**Pull request closed without merging**